emoji/src/components/Form.js

```diff
@@ -0,0 +1,45 @@
+import React, { Component } from 'react';
+import emojiList from './emojiList.json';
+
+class Form extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      filtered: emojiList
+    };
+    this.mapEmoji = this.mapEmoji.bind(this);
+    this.handleChange = this.handleChange.bind(this);
+  }
+
+  handleChange(event) {
+    const { value } = event.target;
+    const filtered = emojiList.filter((emoji) => {
+      if (emoji.keywords.toLowerCase().indexOf(value.toLowerCase()) >= 0 || emoji.title.toLowerCase().indexOf(value.toLowerCase()) >= 0){
+        return emoji.title;
```

Comment:
`Array.prototype.filter` should return values which evaluate to `true` or `false`. Hence, you could simplify your filter callback: `emoji => emoji.keywords.toLowerCase().includes(value.toLowerCase()) || emoji.title.toLowerCase().includes(value.toLowerCase())`